internal/servers/controller/testing.go

```diff
@@ -468,7 +479,7 @@ func NewTestController(t *testing.T, opts *TestControllerOpts) *TestController {
 		opts.Config.Controller = new(config.Controller)
 	}
 	if opts.Config.Controller.Name == "" {
-		opts.Config.Controller.Name, err = base62.Random(5)
+		opts.Config.Controller.Name, err = opts.Config.Controller.InitNameIfEmpty()
```

Comment:
I think this section can be removed right? It'll initialize above now.

Reply:
I've updated the code block above...